source/renderer/app/components/wallet/summary/WalletSummaryAsset.js

```diff
@@ -0,0 +1,213 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import classNames from 'classnames';
+import { get } from 'lodash';
+import SVGInline from 'react-svg-inline';
+import styles from './WalletSummaryAsset.scss';
+import Asset from '../../assets/Asset';
+import AssetAmount from '../../assets/AssetAmount';
+import AssetContent from '../../assets/AssetContent';
+import type { AssetToken } from '../../../api/assets/types';
+import arrow from '../../../assets/images/collapse-arrow-small.inline.svg';
+import warningIcon from '../../../assets/images/asset-token-warning-ic.inline.svg';
+
+const messages = defineMessages({
+  tokenSendButton: {
+    id: 'wallet.summary.asset.tokenSendButton',
+    defaultMessage: '!!!Send',
+    description: 'Send button on Wallet summary assets page',
+  },
+  amountLabel: {
+    id: 'wallet.summary.asset.amountLabel',
+    defaultMessage: '!!!Amount',
+    description: 'Amount label on Wallet summary assets page',
+  },
+  settingsButtonLabel: {
+    id: 'wallet.summary.asset.settings.button.label',
+    defaultMessage: '!!!Settings',
+    description: 'Settings label on Wallet summary assets page',
+  },
+  settingsWarningPopOverAvailable: {
+    id: 'assets.warning.available',
+    defaultMessage:
+      '!!!Recommended configuration for decimal places for this native token is available.',
+    description: 'Asset settings recommended pop over content',
+  },
+  settingsWarningPopOverNotUsing: {
+    id: 'assets.warning.notUsing',
+    defaultMessage:
+      '!!!You are not using the recommended decimal place configuration for this native token.',
+    description: 'Asset settings recommended pop over content',
+  },
+});
+
+type Props = {
+  asset: AssetToken,
+  onOpenAssetSend: Function,
+  onCopyAssetItem: Function,
+  onAssetSettings: Function,
+  anyAssetWasHovered: boolean,
+  isLoading: boolean,
+  assetSettingsDialogWasOpened: boolean,
+};
+
+type State = {
+  isExpanded: boolean,
+};
+
+@observer
+export default class WalletSummaryAsset extends Component<Props, State> {
```

Comment:
Same here, function component could be used instead

Reply:
✅